modules/core/src/lib/layer.js

```diff
@@ -526,14 +526,13 @@ export default class Layer extends Component {
     model.setAttributes(shaderAttributes);
   }
 
-  // Sets the specified instanced picking color to null picking color. Used for multi picking.
-  clearPickingColor(color) {
+  // Sets the picking color at the specified index to null picking color. Used for multi picking.
+  clearPickingColor(objectIndex) {
```

Comment:
Reading this, I wonder if this function could be named better to improve readability. Something like `disablePickingIndex()`?